packages/template-cli/templates/src/ssr/package.json

```diff
@@ -9,15 +9,15 @@
     "lint": "node_modules/.bin/eslint . -f codeframe",
     "prettier": "node_modules/.bin/prettier --ignore-path .gitignore --write \"**/*.+(js|json)\"",
     "validate": "npm run prettier --list-different && npm run lint",
-    "test:pact": "node_modules/.bin/jest --c jest-pact.config.json --rootDir . --runInBand ",
-    "test:pact-publish": "node_modules/.bin/tsc ./pact/pactPublish.ts && node ./pact/pactPublish.js",
-    "test:pact-start-stub": "node_modules/.bin/tsc pact/packStubServer.ts && node pact/packStubServer.js",
-    "test:pact-can-i-deploy": "node_modules/.bin/tsc pact/pactCanIDeploy.ts && node pact/pactCanIDeploy.js",
+    "test:pact": "node_modules/.bin/jest --c jest-pact.config.json --rootDir __tests__ --runInBand",
+    "test:pact-publish": "node_modules/.bin/tsc ./__tests__/pact/pactPublish.ts && node ./__tests__/pact/pactPublish.js",
+    "test:pact-start-stub": "node_modules/.bin/tsc ./__tests__/pact/packStubServer.ts && node __tests__/pact/packStubServer.js",
+    "test:pact-can-i-deploy": "node_modules/.bin/tsc ./__tests__/pact/pactCanIDeploy.ts && node __tests__/pact/pactCanIDeploy.js",
     "test:pact-can-i-deploy-ci": "node_modules/.bin/pact-broker can-i-deploy --pacticipant $PACT_CONSUMER -l --broker-base-url $PACT_BROKER --broker-token $PACT_BEARER_TOKEN",
     "test:cypress:compile": "node_modules/.bin/tsc --project tsconfig.cypress.json",
     "test:cypress:run": "node_modules/.bin/tsc --project tsconfig.cypress.json && node_modules/.bin/cypress run",
     "test:cypress:open": "node_modules/.bin/tsc --project tsconfig.cypress.json && node_modules/.bin/cypress open",
-    "test:cypress": "node_modules/.bin/start-server-and-test start $APP_BASE_URL:$PORT test:cypress:run",
+    "test:cypress": "env CI=true env APP_BASE_URL=http://localhost env PORT=3000 node_modules/.bin/start-server-and-test start $APP_BASE_URL:$PORT test:cypress:run",
```

Comment:
Please pull in from env variables. The port might change.

Reply:
the app will use whatever we tell it to through env vars - it's only a local and in CI task so we can leave it like this otherwise we are abstracting this away in yamls/launch.json etc.. I suggest to users to create an additional task if they want to change the app_base_url or port for whatever reason - though not sure what that might be as it is only a temporary task to spin up app and run tests.
or am I missing something?

Reply:
Yeah but the env variables are pulled in through the config. So by hardcoding here we duplicate the config. Would rather this be pulled in with the env variables that are being used by cypress config, and the webapp it's self.